src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -121,8 +122,10 @@ export const UserInformationContent: React.FunctionComponent<UserInformationCont
         .map((question) => (
           <div key={question.name} className="question-section">
             <div className="question">{question.text}</div>
-            {/* @ts-ignore question.name is a correct index for type User */}
-            <div>{user?.[question.name]}</div>
+            <div>
```

Comment:
Do you need this `div` now that you have the component `<RenderMarkdown/>` ?

Reply:
Probably not, but let me see if it looks different

Reply:
Yeah- it looks the same so I'll delete it